src/events/message/MessageEventHandler.ts

```diff
@@ -42,21 +42,12 @@ export default class MessageEventHandler implements EventHandler<'messageCreate'
 		if ( BotConfig.request.channels && BotConfig.request.channels.includes( message.channel.id ) ) {
 			// This message is in a request channel
 			await this.requestEventHandler.onEvent( message );
-
-			// Don't reply in request channels
-			return;
 		} else if ( BotConfig.request.testingRequestChannels && BotConfig.request.testingRequestChannels.includes( message.channel.id ) ) {
 			// This message is in a testing request channel
 			await this.testingRequestEventHandler.onEvent( message );
-
-			// We want the bot to create embeds in testing channels if someone only posts only a ticket ID
-			// so that people know what the issue is about
 		} else if ( BotConfig.request.internalChannels && BotConfig.request.internalChannels.includes( message.channel.id ) ) {
 			// This message is in an internal channel
 			await this.internalProgressEventHandler.onEvent( message );
-
-			// Don't reply in internal request channels
-			return;
 		}
```

Comment:
I think we should still keep these early returns, just so we don't have to add the configuration options to all the channel descriptions

Reply:
Good Point. I think I did that originally to allow more configuration, but I do think that its unecessary